src/main/java/com/direwolf20/buildinggadgets/common/registry/objects/BGItems.java

```diff
@@ -73,12 +73,16 @@ public static void registerItems(RegistryEvent.Register<Item> event) {
         container.register(event);
     }
 
+    static Item.Properties itemPropertiesWithoutGroup() {
```

Comment:
I'd rather call this `itemProperties` and the other one `itemPropertiesWithGroup`, as well as making the one with a group calling through to this one, instead of calling the constructor directly...?

Reply:
I second this 👍

Reply:
Accepted, But I don't even know why does this exist... XD

Reply:
The effect block still needs to be built but you don't want it to show in the creative tab. Thus without a group

Reply:
I meant just make it don't have an item form. (I'm not sure if you can do that in 1.13, but you can in 1.12)

Reply:
I don't see why you shouldn't be able to do that - it's just that I didn't even consider that to be intendable behaviour and didn't include it in our registration System.
=> adding a boolean hasNoItem with appropiate handling to  BlockBuilder should solve tjat though